ts/features/idpay/initiative/details/screens/OperationsListScreen.tsx

```diff
@@ -0,0 +1,172 @@
+import { OperationTypeEnum as OnboardingOperationTypeEnum } from "../../../../../../definitions/idpay/timeline/OnboardingOperationDTO";
+import { TimelineDTO } from "../../../../../../definitions/idpay/timeline/TimelineDTO";
+import { OperationTypeEnum as TransactionOperationTypeEnum } from "../../../../../../definitions/idpay/timeline/TransactionOperationDTO";
+import { idpayTimelinePaginationGet } from "../store/actions";
+// const mockData: TimelineDTO = {
+//   lastUpdate: new Date(),
+//   operationList: [
+//     {
+//       operationType: TransactionOperationTypeEnum.TRANSACTION,
+//       operationDate: new Date(),
+//       amount: 133.99,
+//       brandLogo:
+//         "https://uat.wisp2.pagopa.gov.it/wallet/assets/img/creditcard/carta_mc.png",
+//       circuitType: "MASTERCARD",
+//       maskedPan: "1234",
+//       operationId: "6"
+//     },
+//     {
+//       operationType: TransactionOperationTypeEnum.REVERSAL,
+//       operationDate: new Date(),
+//       amount: 133.99,
+//       brandLogo:
+//         "https://uat.wisp2.pagopa.gov.it/wallet/assets/img/creditcard/carta_mc.png",
+//       circuitType: "MASTERCARD",
+//       maskedPan: "1234",
+//       operationId: "7"
+//     },
+//     {
+//       operationType: OnboardingOperationTypeEnum.ONBOARDING,
+//       operationDate: new Date(),
+//       operationId: "0"
+//     },
+//     {
+//       operationType: IbanOperationTypeEnum.ADD_IBAN,
+//       operationDate: new Date(),
+//       iban: "IT1234567890123456789012345",
+//       operationId: "1",
+//       channel: "APP"
+//     },
+//     {
+//       operationType: InstrumentOperationTypeEnum.ADD_INSTRUMENT,
+//       operationDate: new Date(),
+//       brandLogo:
+//         "https://uat.wisp2.pagopa.gov.it/wallet/assets/img/creditcard/carta_mc.png",
+//       maskedPan: "1234",
+//       operationId: "2",
+//       channel: "APP"
+//     },
+//     {
+//       operationType: InstrumentOperationTypeEnum.ADD_INSTRUMENT,
+//       operationDate: new Date(),
+//       brandLogo:
+//         "https://uat.wisp2.pagopa.gov.it/wallet/assets/img/creditcard/carta_visa.png",
+//       maskedPan: "1234",
+//       operationId: "3",
+//       channel: "APP"
+//     },
+//     {
+//       operationType: InstrumentOperationTypeEnum.DELETE_INSTRUMENT,
+//       operationDate: new Date(),
+//       brandLogo:
+//         "https://uat.wisp2.pagopa.gov.it/wallet/assets/img/creditcard/carta_mc.png",
+//       maskedPan: "1234",
+//       operationId: "4",
+//       channel: "APP"
+//     },
+//     {
+//       operationType: InstrumentOperationTypeEnum.DELETE_INSTRUMENT,
+//       operationDate: new Date(),
+//       brandLogo:
+//         "https://uat.wisp2.pagopa.gov.it/wallet/assets/img/creditcard/carta_visa.png",
+//       maskedPan: "1234",
+//       operationId: "5",
+//       channel: "APP"
+//     }
+//   ]
+// };
```

Comment:
![b5OjfFNDD7AG9iPOJ_e5os-HnYtZPwe-3icEsevZGHQ - dimensioni piccole](https://user-images.githubusercontent.com/6160324/212102792-e01f942c-b36d-42cf-92ea-7fd7721e3265.jpeg)

Jokes apart: if you don't need this code just throw it away :)